### DISCUSSION TITLE
[Discuss] Will nerdctl be merged into ctr in the future?

### DISCUSSION THREAD

**Discussion by maxsxu** [containerd/nerdctl#763] [Q&A] [answered]
I've been using `nerdctl` for a while, but one thing has been bothering me: What is the future plan of `nerdctl`?

As we all know, the [ctr](https://github.com/containerd/containerd/tree/master/cmd/ctr) has always been a part of `containerd`. Now the `nerdctl` works like another real containerd CLI "`ctr`" (as this name points), which drives people crazy actually sometimes.

I guess two things will happen:

- `nerdctl` and `ctr` will be merged into a single project, which named `ctr` (I love this one)
- `nerdctl` and `ctr` will coexist for a long time

How do you think?

**Comment by AkihiroSuda** [comment, marked as answer]
No, nerdctl and ctr will remain to coexist as separate CLIs. 

ctr is just a debugging tool. Not a "real" CLI.

**Comment by afbjorklund** [comment]
And of course there is also `buildctl`, which nerdctl uses.
